### PR TITLE
Latest from unstable

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -97,6 +97,7 @@
     <resource-file src="src/ios/resources/SalesforceSDKAssets.xcassets" />
 
     <!-- Libraries -->
+    <source-file src="src/ios/frameworks/libSalesforceAnalytics.a" framework="true" />
```
Comment (bhariharan) [inline plugin.xml]: Added dependency on `SalesforceAnalytics` here.

File: tools/update.sh [hunk 2]
```diff
@@ -175,6 +175,8 @@ create_android_dirs()
 copy_ios_sdk()
 {
     echo "*** iOS ***"
+    echo "Copying SalesforceAnalytics library"
```
Comment (bhariharan) [inline tools/update.sh]: Adding `SalesforceAnalytics` library.

File: tools/postinstall-android.js [hunk 3]
```diff
@@ -58,22 +58,26 @@ var libProjectRoot = path.join('plugins', 'com.salesforce', 'src', 'android', 'l
...
 console.log('Moving Salesforce libraries to the correct location');
+shelljs.cp('-R', path.join(libProjectRoot, 'SalesforceAnalytics'), appProjectRoot);
 shelljs.cp('-R', path.join(libProjectRoot, 'SalesforceSDK'), appProjectRoot);
 shelljs.cp('-R', path.join(libProjectRoot, 'SmartStore'), appProjectRoot);
 shelljs.cp('-R', path.join(libProjectRoot, 'SmartSync'), appProjectRoot);
 shelljs.cp('-R', path.join(libProjectRoot, 'SalesforceHybrid'), appProjectRoot);
 
 console.log('Fixing Gradle dependency paths in Salesforce libraries');
 var oldCordovaDep = "compile project\(\':external:cordova:framework\'\)";
+var oldSalesforceAnalyticsDep = "compile project\(\':libs:SalesforceAnalytics\'\)";
```
Comment (bhariharan) [inline tools/postinstall-android.js]: Postinstall script updates for the new library.

**Comment by bhariharan** [comment]
Everything else is auto updated from the script.

**Comment by khawkins** [comment]
👍

**Pull request merged**